### DISCUSSION TITLE
Krosh's fireball ignoring fire resist in Gruul's Lair

### DISCUSSION THREAD

**Discussion by TelumehtarCC** [chromiecraft/chromiecraft#7287] [Suggestions]
Why was this added in a recent patch? There's so little purpose for the fire-resist badge gear as is. I feel like it further invalidated the value of 40 badges per item for that fire resist gear set. 

Please consider reverting that change.

**Comment by heyitsbench** [comment]
Because it's Blizzlike.

**Comment by BeefSteaker** [comment]
Unfortunately if it is blizzlike then it's unlikely to be changed. If you can find a source that it's not blizzlike then you can make a bug report.

**Comment by TelumehtarCC** [comment]
THX.
